MCDTool.egg-info/PKG-INFO

```diff
@@ -0,0 +1,14 @@
+Metadata-Version: 2.1
+Name: MCDTool
+Version: 0.0.1
```

Comment:
If you use semantic versioning, we should also define what would constitute version 1.0 (the MVP).

Reply:
?